contract/package.json

```diff
@@ -5,17 +5,6 @@
   "description": "Agoric Orchestration Contract",
   "type": "module",
   "scripts": {
-    "start:docker": "docker compose up -d",
-    "docker:logs": "docker compose logs --tail 200 -f",
-    "docker:bash": "docker compose exec agd bash",
-    "docker:make": "docker compose exec agd make -C /workspace/contract",
-    "make:help": "make list",
-    "start": "make clean start",
```

Comment:
likewise `yarn start` should give a clue about what to do instead.

Reply:
`yarn start` now deploys contract on chain, i.e., `make e2e`
Updated the rest same as above.